_pages/how-we-work/tools/github.md

```diff
@@ -54,20 +54,20 @@ Go to the [18F people page](https://github.com/orgs/18F/people). Click where it
 
   - To help you not commit sensitive information to Github, [please read about Git
     Seekrets]({{site.baseurl}}/sensitive-information#git-seekret).
 
-- **Ask Infrastructure before integrating a service with GitHub.** Many websites offer the option to "Sign in with GitHub" and may further request permission to access your "personal user data." Providing this level of access can not only share your public or private email address, but it can also grant the ability to access 18F's private repositories. For this reason, we ask that all organization members refrain from authorizing integrations and request any desired integrations through an [Infrastructure issue](https://github.com/18F/infrastructure).
+- **Ask TTS Tech Portfolio before integrating a service with GitHub.** Many websites offer the option to "Sign in with GitHub" and may further request permission to access your "personal user data." Providing this level of access can not only share your public or private email address, but it can also grant the ability to access 18F's private repositories. For this reason, we ask that all organization members refrain from authorizing integrations and request any desired integrations through an TTS Tech Portfolio issue.
 
-- **Ask Infrastructure before creating private repositories.** We pay GitHub for the ability to create private repositories and need to bill clients for repositories created on their behalf. Before you do anything, drop into [#admins-github](https://gsa-tts.slack.com/messages/admins-github) and explain what you'd like to do and why. Be sure to include a link in your repo README to a document that explains why it is private. (See the **Exceptions** section of the [18F Open Source policy](https://18f.gsa.gov/open-source-policy/).)
+- **Ask TTS Tech Portfolio before creating private repositories.** We pay GitHub for the ability to create private repositories and need to bill clients for repositories created on their behalf. Before you do anything, drop into [#admins-github](https://gsa-tts.slack.com/messages/admins-github) and explain what you'd like to do and why. Be sure to include a link in your repo README to a document that explains why it is private. (See the **Exceptions** section of the [18F Open Source policy](https://18f.gsa.gov/open-source-policy/).)
```

Comment:
Is this still true? I have some vague recollection that Github changed pricing structures and we're not billed per-private-repo anymore, but I could also be 100% wrong about that.

On the other hand, even if it has changed, we default to working in the open, so it still might be worth discouraging private repos.